galactus.theme

```diff
@@ -23,9 +24,12 @@ function galactus_page_attachments_alter(array &$page) {
   }
 
   // Switch between minimal and full CLF libraries.
-  if ($use_minimal_clf) {
+  if ($clf_source == 'min' || $use_minimal_clf) {
```

Comment:
Explanation: check for either legacy setting for minimal CLF or the new theme source setting to load the CDN version of the minimal CLF